app/controllers/projects_controller.rb

```diff
@@ -3,10 +3,32 @@ class ProjectsController < ApplicationController
   before_action :login_required, only: [:new]
 
   def new
+    @project = Project.new
   end
 
   def index
     @projects = Project.all
   end
 
+  def create
+    @project = Project.new(project_params)
+
+    @project.submitter_id = current_user.id
```

Comment:
A taste thing, but I prefer to assign high level objects instead of IDs for AR associations. In this case: `@project.submitter = current_user`